code/modules/unit_tests/area_contents.dm

```diff
@@ -0,0 +1,29 @@
+/// Verifies that an area's perception of their "turfs" is correct, and no other area overlaps with them
+/// Quite slow, but needed
```

Comment:
If it's slow, then put it in a later priority (probably doesn't need to be its own, just a "slow" priority is fine)